CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+cmake_minimum_required(VERSION 3.15...3.26)
+project(python-casacore)
+
+find_package(
+  Python
+  COMPONENTS Interpreter Development.Module
+  REQUIRED)
+
+# Find Casacore and its dependencies
+find_package(PkgConfig REQUIRED)
+pkg_check_modules(Casacore REQUIRED casacore)
```

Comment:
This assumes casacore 3.5, which is not yet in ubuntu 22.04. I'd prefer having a fallback based on a FindCasacore.cmake for people who are not on casacore 3.5.

Reply:
Replaced `pkg-config` stuff with good-ole CMake modules. No need to do fancy stuff :)